src/braket/tasks/local_quantum_task.py

```diff
@@ -42,8 +42,12 @@ def id(self) -> str:
         return str(self._id)
 
     def cancel(self) -> None:
-        """Cancel the quantum task."""
-        raise NotImplementedError("Cannot cancel completed local task")
+        """Attempt to cancel the quantum task.
```

Comment:
nit: if we are always writing `pass` and the task is expected to be complete, attempt has the wrong implication.